src/interpreter/vm.h

```diff
@@ -186,7 +197,7 @@ namespace verona::interpreter
        *
        * This is unused in the lowest frame, as exiting that frame halts the VM.
        */
-      size_t return_address;
+      size_t ip;
```

Comment:
Does this comment need updating for `Halt` frames?

Reply:
Done.